org-formation/300-account-defaults/_tasks.yaml

```diff
@@ -53,3 +53,16 @@ ItKmsKey:
     IncludeMasterAccount: true
     Account: '*'
     Region: !Ref primaryRegion
+
+VpcPeeringRequest:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.2.12/templates/VPC/vpc-peering-request.yaml
+  StackName: !Sub '${resourcePrefix}-vpc-peering-request'
+  StackDescription: 'Authorize VPC peering requests'
+  Parameters:
+    VpcPeeringRequesterAwsAccountId: '745159704268'
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: false
+    Account:
+      - !Ref NlpSandboxAccount
```

Comment:
Where's this defined? Should it be in ../_parameters (and add nextflow* there)?

Reply:
it's in organization.yaml

Reply:
I see, could you add the nextFlow accounts here then?
- !Ref WorkflowsNextflowDevAccount
- !Ref WorkflowsNextflowProdAccount

Reply:
done